bedrock/firefox/templates/firefox/os/index-new.html

```diff
@@ -0,0 +1,244 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/base-resp.html" %}
+
+{% add_lang_files "firefox/os/devices" %}
+
+{% block page_title_prefix %}{% endblock %}
+{% block page_title %}{{_('Firefox OS — The Adaptive Phone — Great Smartphone Features, Apps and More')}}{% endblock %}
+{% block page_desc %}{{_('Firefox OS smartphones give you great adaptive features and apps that let you live every moment to its fullest and build a brighter future for the Web.')}}{% endblock %}
+
+{% block body_id %}firefox-os{% endblock %}
+{% block body_class %}firefox-os{% endblock %}
+
+{# Do not load CSS bundle specified in base template (duplicate CSS). #}
+{% block site_css %}{% endblock %}
+
+{% block extrahead %}
+  {{ super() }}
+  {% stylesheet 'firefox_os_new' %}
+<!--[if lt IE 9]>
+  {% stylesheet 'firefox_os_ie' %}
+<![endif]-->
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block site_header_unwrapped %}
+  {% include 'firefox/includes/family-nav.html' %}
+{% endblock %}
+
+{% block content %}
+  <main role="main">
+    <section class="fxos-hero-space">
+      <div class="content-wrapper">
+        <div class="phone">
+          <div class="fxos-cta">
+            <h1><span>{{_('Firefox OS')}}</span></h1>
+            <p class="tagline">{{_('Designed to give you just what you need, just when you need it.')}}</p>
+
+            <div id="primary-cta-signup" class="signup hidden">
+              <a href="#email-form-wrapper" class="cta-button get-newsletter newsletter-signup-toggle" role="button" aria-haspopup="true">{{_('Sign me up')}}</a>
+              <p class="signup-txt">{{_('Sign up for our monthly newsletter and get the latest on Firefox OS')}}</p>
+            </div>
+            <a id="primary-cta-phone" href="#get-device" class="cta-button get-phone hidden" role="button" aria-haspopup="true">{{_('Get a phone')}}</a>
+          </div>
+
+          <div href="#" class="phone-container">
+            <img src="{{ static('img/firefox/os-new/device_static.png') }}" width="1030" height="782" alt="{{_('Firefox OS device illustration')}}" />
+
+            <!-- device demo triggers, not going live on March 1, so commenting out for now
```

Comment:
Might be preferable here to use a jinja coment i.e `{# #}` so this doesn't get output in the HTML.